site/components/DisruptionDateTimeInfo.tsx

```diff
@@ -29,12 +28,9 @@ export const DisruptionsDateTimeInfo = ({
             <DateSelector
                 errors={[]}
                 startOrEnd="start"
-                inputs={{
-                    dayInput: isDisruptionValidity ? inputs.validityStartDateDay : inputs.publishStartDateDay,
-                    monthInput: isDisruptionValidity ? inputs.validityStartDateMonth : inputs.publishStartDateMonth,
-                    yearInput: isDisruptionValidity ? inputs.validityStartDateYear : inputs.publishStartDateYear,
-                }}
-                type={isDisruptionValidity ? "validity" : "publish"}
+                input={!!inputs.validityStartDate ? new Date(inputs.validityStartDate) : null}
+                disabled={false}
+                isPublishInfo={false}
```

Comment:
This needs to be defined by `isDisruptionValidity`

Reply:
added